src/pipes/array/implode

```diff
@@ -0,0 +1,29 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+// does the same thing as implode in PHP
+// concat elements of array and return string
+@Pipe({
+  name: 'implode'
+})
+export class ImplodePipe implements PipeTransform {
+
+  transform(value: any[], delimiter: any, objectKey?: string): string {
+
+    if(!Array.isArray(value)){
+      let _type = typeof value;
+      throw new Error(`Implode expect an array ${_type} given!`);
```

Comment:
change to `throw new Error(`Implode expected an array but ${typeof value} given.`)`